apps/frontend/src/app/2_molecules/DebugContent.tsx

```diff
@@ -256,7 +256,7 @@ export const DebugContent = () => {
 
       <h3>An example of the Transaction Frame</h3>
       <br />
-      <TransactionHistoryFrame />
+      <TransactionHistoryFrame account={wallets[0]?.accounts[0]?.address} />
```

Comment:
```suggestion
      <TransactionHistoryFrame account={account} />
```
Use `account` from `useAccount` hook.